src/components/Layout.tsx

```diff
@@ -20,13 +20,15 @@ export default function Layout({ children }: LayoutProps) {
                         }}
                         className="flex items-start h-full bg-[url('/pixelpokeball.png')] bg-[0_0] [aspect-ratio:75/100] hover:animate-pokeballopening"
                     ></div>
-                    <Image
-                        src="/pokacid-logo.png"
-                        alt="pokacid logo"
-                        width={342}
-                        height={100}
-                    />
-                    <button className="m-2.5">Light/Dark</button>
+                    <Link href="/" className="my-2">
+                        <Image
+                            src="/pokacid-logo.png"
+                            alt="pokacid logo"
+                            width={288}
+                            height={84}
+                        />
+                    </Link>
+                    <div></div>
```

Comment:
Having this empty placeholder element results in the logo not actually being centered, but being shifted to the right a little. One option would be to give the placeholder element the same width as the opposing item (the pokeball).

Another approach could be to make the header a grid instead of a flex element, where the left and right column have the same width (1fr) and the middle one has an auto width.